scripts/azureml-assets/azureml/assets/config.py

```diff
@@ -798,6 +798,10 @@ def __lt__(self, other) -> bool:
         # Compare versions using packaging's version object
         return version.parse(self.version) < version.parse(other.version)
 
+    def __hash__(self) -> int:
+        """Hash an AssetConfig object."""
+        return hash((self.type.value, self.name, self.version))

```

Comment:
why two set of parentheses ther?

Reply:
To create a tuple, because `hash()` accepts an object.